UI/OnScreenDisplay.cpp

```diff
@@ -28,18 +28,33 @@ void OnScreenMessagesView::Draw(UIContext &dc) {
 		float alpha = (iter->endTime - now) * 4.0f;
 		if (alpha > 1.0) alpha = 1.0f;
 		if (alpha < 0.0) alpha = 0.0f;
+		dc.SetFontScale(1.0f, 1.0f);
 		// Messages that are wider than the screen are left-aligned instead of centered.
+
+		int align = 0;
+		// Simple heuristic: Draw messages that contain line breaks with ASCII_FONT (likely debug output)
```

Comment:
Hm, I could imagine some language like German wanting to use two lines for some message...

-[Unknown]

Reply:
Just so happens that German works fine with the normal font, I think...

Anyway, yeah this is not ideal, should do something better.